pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
I will look into this. I believe @jeremybmerrill one time mentioned that Unix systems require a terminal install command for a `tesseract` package, but I'll investigate this and find the fix.